bedrock/settings/base.py

```diff
@@ -1225,6 +1226,8 @@ def get_default_gateway_linux():
     "cdn.cookielaw.org",
     "privacyportal.onetrust.com",
     FXA_ENDPOINT,
+    "com-getpocket-prod1.mini.snplow.net",  # Dev Pocket Snowplow
```

Comment:
where do we use dev snowplow? Can this be appended only in those situations? e.g.

```python
if DEV:
    CSP_CONNECT_SRC.append("com-getpocket-prod1.mini.snplow.net")
```

Reply:
It's the local "connectorUrl" used in the `snowplow('new tracker', 'sp', connectorUrl...` call. Makes sense to only append when needed. Will change.

Reply:
Ah, slight block on the change to pre-commit config, I'm now seeing a bunch of flake8 failures in the `settings/base.py` file:
`flake8...................................................................Failed
- hook id: flake8
- exit code: 1

bedrock/settings/base.py:937:1: E402 module level import not at top of file
bedrock/settings/base.py:937:1: F401 '.appstores.ADJUST_FIREFOX_URL' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.ADJUST_FOCUS_URL' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.ADJUST_KLAR_URL' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.ADJUST_LOCKWISE_URL' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.ADJUST_POCKET_URL' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.AMAZON_FIREFOX_FIRE_TV_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.APPLE_APPSTORE_COUNTRY_MAP' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.APPLE_APPSTORE_FIREFOX_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.APPLE_APPSTORE_FOCUS_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.APPLE_APPSTORE_KLAR_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.APPLE_APPSTORE_LOCKWISE_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.APPLE_APPSTORE_POCKET_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_FIREFOX_BETA_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_FIREFOX_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_FIREFOX_LITE_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_FIREFOX_NIGHTLY_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_FIREFOX_SEND_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_FOCUS_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_KLAR_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_LOCKWISE_LINK' imported but unused
bedrock/settings/base.py:937:1: F401 '.appstores.GOOGLE_PLAY_POCKET_LINK' imported but unused
1     E402 module level import not at top of file
21    F401 '.appstores.ADJUST_FIREFOX_URL' imported but unused`

I may revert the config change and leave it for a separate PR. Will take this up again tomorrow.